examples/select_streaming_on_data.ts

```diff
@@ -0,0 +1,32 @@
+import type { Row } from '../src'
+import { createClient } from '../src'
+
+/**
+ * Can be used for consuming large datasets for reducing memory overhead,
+ * or if your response exceeds built in Node.js limitations,
+ * such as 512Mb for strings
```

Comment:
Let's add a link to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/length
> In V8 (used by Chrome and Node), the maximum length is 229 - 24 (~1GB). On 32-bit systems, the maximum length is 228 - 16 (~512MB).